README.md

```diff
@@ -1,74 +1,142 @@
-**UNDER DEVELOPMENT**
-
 # SOLVCON Development Environment
 
-This project provides a reference runtime environment for developing SOLVCON
-and related projects to reduce unnecessary misunderstandings.  It supports the
-following features:
+**UNDER DEVELOPMENT**
 
-1. Support ubuntu and macOS.
-2. Manage dependencies in the source-code level.  It may build all dependencies
-   from source and retain the source code, so that a debugger can read it.
-3. Allow building a selected set of packages from repository / master.  The
-   minimal set includes cpython, numpy, and pybind11.
-4. Allow multiple copies of the runtime environment and build at the same time.
-   At least support two flavors: release and debug.
+This project is a set of (Bourne again) shell scripts to align the environment
+for developing SOLVCON and related projects in (Ubuntu) Linux and macOS.  The
+features include:
 
-The environment consists of scripts for building dependencies and managing
-them.  More information will be added as the project becomes more complete.
+1. Build dependencies from source and retain the source code for debugger.
+2. Unprivileged installation.  No root permission is required.
+3. Switchable environments.
+4. Download source code from a repository or a tarball.
 
-## Pre-requisite
+# Installation
 
-By using the following command to setup all necessary environment variables:
+Clone the repository to where you want to install, usually somewhere under your
+home directory (e.g., `~/devenv`):
 
+```bash
+git clone git@github.com:solvcon/devenv.git ~/devenv
 ```
-source scripts/init
+
+(Or use https:)
+
+```bash
+git clone https://github.com/solvcon/devenv ~/devenv
 ```
 
-## Advanced Configuration
+## Enabling
 
-* Customized flavors directory path: You may define the `DEVENVFLAVORROOT`
-environment variable to point to the flavors root path where you want.
+DEVENV requires bash and you need to turn it on in the shell:
 
-## Usage
+```bash
+source ~/devenv/scripts/init
+```
 
-By invoking this one command should bring you the whole SOLVCON stack:
+It can be done automatically if you add the following line in the startup file
+(`~/.bashrc` or `~/.bash_profile`):
 
 ```bash
-./bin/build-application-solvcon-devenv.sh
+if [ -f ~/devenv/scripts/init ]; then source ~/devenv/scripts/init; fi
 ```
 
-You may use the management tool `devenv` to build the target package you need
-from source.  By placing the command to see the details of usage:
+# Usage
```

Comment:
"Usage" is a new section that reorganized information from the old README and added new contents.